model/model_training/tools/export.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import sys
+
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("model_name", type=str, help="checkpoint path or model name")
+    parser.add_argument("--dtype", type=str, default="float16", help="float16, float32")
+    parser.add_argument("--hf_repo_name", type=str, help="Huggingface repository name (used with --push)")
+    parser.add_argument("--auth_token", type=str, help="")
+    parser.add_argument("--output_folder", type=str, help="output folder path (used with)")
+    parser.add_argument("--max_shard_size", type=str, default="10GB")
+    parser.add_argument("--cache_dir", type=str)
+    return parser.parse_args()
+
+
+def main():
```

Comment:
This file can be deleted. The actual export tool is called export_model.py in the same directory ;-).

Reply:
Not sure why this was still here...